HelpSource/Classes/Env.schelp

```diff
@@ -8,11 +8,32 @@ An Env is a specification for a segmented envelope. Envs can be used both server
 
 An Env can have any number of segments which can stop at a particular value or loop several segments when sustaining. It can have several shapes for its segments.
 
+The envelope is conceived as a sequence of emphasis::nodes:: (not to be confused with a synthesis-Node) each of which has three parameters: a target level, a time duration from the previous node, and a shape. The three parameters for each node are kept in separate arrays as explained below.
+
 code::
-Env.new([0, 1, 0.9, 0], [0.1, 0.5, 1],[-5, 0, -5]).plot;
+Env.new(levels: [0, 1, 0.9, 0], times: [0.1, 0.5, 1], curve: [-5, 0, -5]).plot;
 ::
 
-The envelope is conceived as a sequence of emphasis::nodes:: (not to be confused with a synthesis-Node) each of which has three parameters: a target level, a time duration from the previous node, and a shape. The three parameters for each node are kept in separate arrays as explained below.
+In this envelope, there are three emphasis::nodes:: :
```

Comment:
Responding to your comments on Slack. The first value in an Env's levels array is a node because it is the node at index 0. In this example, I am setting the releaseNode to 0 which will cause the evnelop to stay on its inital value untill the gate is closed:

```
(
{
	EnvGen.kr(
		// sustains at level 0 until gate is closed for the first time
		Env([0, 0.2, -0.2, 0], 1e-2, releaseNode: 0), 
		gate: Trig.kr(Impulse.kr(4), dur: 0.2)
	);
}.plot(duration: 1);
)
```